staging/src/k8s.io/apiserver/pkg/admission/configuration/validating_webhook_manager.go

```diff
@@ -76,7 +77,8 @@ func (v *validatingWebhookConfigurationManager) updateConfiguration() {
 		utilruntime.HandleError(fmt.Errorf("error updating configuration: %v", err))
 		return
 	}
-	v.configuration.Store(mergeValidatingWebhookConfigurations(configurations))
+	c := mergeValidatingWebhookConfigurations(configurations)
+	v.configuration.Store(webhook.ConvertToValidatingWebhookAccessors(c.Webhooks))
```

Comment:
nit: it looks like this is the only call site to ConvertToValidatingWebhookAccessors and mergeValidatingWebhookConfigurations, could delete ConvertToValidatingWebhookAccessors and inline the wrapping into mergeValidatingWebhookConfigurations and have it return []webhook.WebhookAccessor. would be simpler and save list allocs.

Reply:
same applies for ConvertToMutatingWebhookAccessors / mergeMutatingWebhookConfigurations

Reply:
Applied. Cleaned things up nicely.